jenkins_jobs/trigger_weekly_host_os_build/script.sh

```diff
@@ -77,7 +77,7 @@ create_release_notes() {
     python host_os.py \
            --verbose \
            build-release-notes \
-               --packages-metadata-repo-url "$VERSIONS_REPOSITORY_URL" \
+               --packages-metadata-repo-url "$VERSIONS_PUSH_URL" \
```

Comment:
This doesn't seem right to me. The push URL is the forked repo of the user doing the update (the bot in the general case), I don't think we should be reading from this repo, it might not even be up-to-date.

update-versions uses the upstream repo for fetching https://github.com/olavphibm/infrastructure/blob/a73ee0d6ec53074af3ea7d0d4fc6380532b9d25d/jenkins_jobs/trigger_weekly_host_os_build/script.sh#L68